docs/design-proposals/multi-tenancy/multi-tenancy-network.md

```diff
@@ -283,7 +283,158 @@ kube-dns-{network} shall be managed by Endpoints controller just like a regular
 
 __EndpointSlices__, the new type introduced in k8s v1.17, is out of current scope.
 
-### Ingress/egress/network policies
+### Network Policy
+
+Network policies, expressed as intents, are used to enforce desired network security within Arktos. Without any network policy in place, Arktos allows any pod to access other pods, across all tenants and namespaces for a flat typed network environment. VPC typed network environment, on the other hand, does provide strong tenant level network security even in the absence of any network policy in place.
+
+Existing k8s network policy specifications provide support for enforcing tenant scoped network security. Allowing access across tenant boundary is currently not supported within Arktos.
```

Comment:
"allowing access across tenant boundary is currently not supported within Arktos".  Is this related to network provider instead of network policy?  For example, do we allow cross-tenant access when we use flat network provider?

Reply:
disallowing access across tenant is based on considerations:
1. current k8s network policy syntax is short of tenant related selector; 
2. no existing k8s network policy rule enforcer understands tenant and process properly;

It is progamatic to start from the basic and most strict case. After we gain more understandings along the way, we can consider the support of cross-tenant access -- probably by extending (even crafting new syntax centered around network, if it is thought boldly) the network policy syntax.